content/integrations/azure.md

```diff
@@ -233,13 +244,28 @@ Naviate to the [Azure VM Default Dashboard][6] to see this dashboard populate wi
 
 Learn more about how to monitor Azure VM performance metrics with [our series of posts](https://www.datadoghq.com/blog/how-to-monitor-microsoft-azure-vms/). We detail the key performance metrics, how to collect them, and how to use Datadog to monitor Azure VMs.
 
+### Configuration
+{: #configuration}
+
+Optionally, you can limit the Azure VMs that are pulled into Datadog by entering tags in the "Optionally filter to VMs with tag" textbox. This comma separated list of tags (in the form 'key:value') defines a filter that we will use when collecting metrics from Azure VMs. Wildcards, such as '?' (for single characters) and '*' (for multiple characters) can also be used. Only VMs that match one of the defined tags will be imported into Datadog. The rest will be ignored.
```

Comment:
Question here (I might be wrong): is it possible to give multiple tag values on a single tag? e.g. `datacenter:dc1,dc2` or must it be `datacenter:dc1,datacenter:dc2`?

In any case, we should give some examples of provided tags, including with a comma-separated list. Just saying `key:value` is good, but examples are even better.